entrypoint.sh

```diff
@@ -70,8 +70,18 @@ if [ "$1" = "kubelet" ]; then
 
     CGROUPDRIVER=$(/opt/rke-tools/bin/docker info | grep -i 'cgroup driver' | awk '{print $3}')
 
+    if [ "${RKE_KUBELET_CRIDOCKERD}" == "true" ]; then
+        /opt/rke-tools/bin/cri-dockerd &
+    fi
+    # wait for cri-dockerd to start as kubelet depends on it
+    echo "Sleeping 10 waiting for cri-dockerd to start"
```

Comment:
No real health check because of GRPC, this is taken from https://github.com/kubernetes-sigs/cri-tools/blob/80414ae7c6f2e82216039be756ed4a4221e6068f/hack/run-dockershim-critest.sh#L44